pele_platform/Frag/libraries.py

```diff
@@ -1,36 +1,54 @@
 import glob
 import os
 import subprocess
+import shutil
+import tempfile
 
+from rdkit import Chem
 from pele_platform.constants import constants as cs
 import pele_platform.Utilities.Helpers.helpers as hp
 
 OUTPUT = "input.conf"
 
+
+def getSymmetryGroups(mol):
+    """
+    Computes the symmetry class for each atom and returns a list with the idx of non-symmetric atoms.
+    """
+    rank = {} 
+    symmetryList=[] 
+    symmetryRankList=[]
+    counter=0
+    
+    for atom in mol.GetAtoms():
+        rank[atom.GetIdx()] = list(Chem.CanonicalRankAtoms(mol,breakTies=False))[counter]
+        counter += 1
+    
+    for idx, symmetryRank in rank.items():
```

Comment:
Variable names such as `symmetryRank` should also fulfil PEP8 standards: lowercase letters with underscores (`symmetry_rank`). Apart from that, your code looks great! :)

Reply:
The same applies to `symmetryList`.